### PR TITLE
304 update ruby and rails in test suite

### PR DESCRIPTION
**Pull request by oraposa**
This pull request upgrades Tenejo to Ruby 2.7.4 and Ruby on Rails to v. 5.2.6. This includes changes to numerous files as a result of the change in Rubocop version (see below) and items e.g., .ruby-version, to placate CircleCI. The suite upgrade requires local upgrades to Ruby and bundler.

Other updated dependencies include Sidekiq to 5.2.7 (formerly 5.1.3), bixby to  3.0.2 (2.0.0.pre.beta1), which upgrades rubocop to version 0.85.1 (0.63). 

#### To test:
- install ruby 2.7.4 using your preferred ruby installer
- `gem install bundler:2.2.28`
- `git checkout 304-update_ruby_and_rails_in_test_suite_rubocop_corrected`
-  `bundle install`
- migrate test db
-- `bundle exec rails db:migrate RAILS_ENV=test`
- start tenejo test suite as directed in [tenejo README](https://github.com/curationexperts/tenejo/blob/main/README.md).
- run tests
-- `bundle exec rspec spec/`
-- verify tests run as expected.

Done.

### CODE REVIEW

File: .circleci/.ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6.6
```
Comment (maxkadel) [inline .circleci/.ruby-version]: This is confusing, given the ruby version in the .circleci/config.yml (2.7.4)? Do we need this file?

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1,2 @@
+2.7.4
```
Comment (maxkadel) [inline .ruby-version:1]: Do we need this file?

**Review by maxkadel — CHANGES_REQUESTED** [review]
I'm not sure we need either of the .ruby-version files, but if we do need them, they should match (the .circleci one doesn't match)

File: .ruby-version (hunk 2 above)
Comment (oraposa) [reply .ruby-version:1]: Removed.

File: .circleci/.ruby-version (hunk 1 above)
Comment (oraposa) [reply .circleci/.ruby-version]: removed.

File: .ruby-version (hunk 2 above)
Comment (maxkadel) [reply .ruby-version:1]: Is it, though?

**Review by maxkadel — APPROVED** [review]

**Pull request merged**